media/static-bundles.json

```diff
@@ -907,6 +914,17 @@
       ],
       "name": "firefox_whatsnew_77_zhCN"
     },
+    {
+      "files": [
+        "js/base/uitour-lib.js",
```

Comment:
As of https://github.com/mozilla/bedrock/pull/8914 this lib is now included in our common bundles. so you should be able to remove it here.